source/app/service-providers/commands/dir-rescan.ts

```diff
@@ -25,7 +25,8 @@ export default class DirRescan extends ZettlrCommand {
     * @param {String} evt The event name
     * @param  {Object} arg The path of the descriptor
     */
-  async run (evt: string, arg: any): Promise<void> {
-    await this._app.workspaces.rescanForDirectory(arg.path)
+  async run (event: string, _arg: any): Promise<void> {
+    // DEBUG: DEPRECATED
+    throw new Error('rescanForDirectory not re-implemented')
```

Comment:
This hasn't been functional since ages and nobody has complained. That's only neessary for a very small set of people who have files on a thumb drive, so that they can plug and unplug it. OOS for this PR.